test/e2e/services.go

```diff
@@ -73,6 +105,64 @@ func startServicesForLogs(t *testing.T, e e2e.Environment) (
 	return loki.InternalEndpoint("http"), loki.Endpoint("http")
 }
 
+func startServicesForTraces(t *testing.T, e e2e.Environment) (otlpGRPCEndpoint string, jaegerHttpEndpoint string) {
+	jaeger := e.Runnable("jaeger").
+		WithPorts(
+			map[string]int{
+				"jaeger.grpc": 14250, // Receives traces
+				"grpc.query":  16685, // Query
+				"http.query":  16686, // Query
+			}).
+		Init(e2e.StartOptions{Image: "jaegertracing/all-in-one:1.31"})
+
+	// Warn if a YAML change introduced a tab character
+	if strings.ContainsRune(otelConfig, '\t') {
+		t.Errorf("Tab in the YAML")
+	}
+
+	config := strings.Replace(otelConfig,
```

Comment:
I think we could move the creation of the yaml config file to https://github.com/observatorium/api/blob/main/test/e2e/configs.go and add a function like `createTracesYAML` or something similar - what do you think?
So [this](https://github.com/observatorium/api/blob/ca9ffa0586a31db2ef842713ccfc1363fc783a96/test/e2e/services.go#L37-L63) could also be moved there